Lib/mimetypes.py

```diff
@@ -90,6 +90,8 @@ def add_type(self, type, ext, strict=True):
         list of standard types, else to the list of non-standard
         types.
         """
+        if not type and ext in self.types_map[strict]:
+            type = self.types_map[strict][ext]
```

Comment:
I haven't thought this through, but is this better than just ignoring calls where `type` is empty?

Reply:
I think this is a great idea. If it is supported in mimetypes, mapping a missing value in the `.webp` registry entry to a default value is necessary. After all, this approach ensures consistency in the program. Most users are unlikely to pay much attention to any specific attributes in the registry.

> but is this better than just ignoring calls where type is empty?

If empty values are ignored, does that imply the attribute is unsupported? This doesn’t seem to align with user expectations. Typically, they would expect it to return `('image/webp', None)` rather than exhibit any other behavior.
(I hope I haven’t misunderstood what you mentioned about ignoring empty values.)

Reply:
I mean like this:
```suggestion
        if not type:
            return
```

This function `add_type` gets called when we are reading each type from the registry. _Missing_ values don't get read at all, so they don't matter here, but _empty_ values do get read and passed to this function. But we don't want to overwrite our default with an empty value, so we should ignore it.

Reply:
It seems that this would be more concise rather than remapping.